wordpress-mobile/WordPress-iOS
# Uses reader Topics to populate the new Reader Tab Bar

Ref #13580

To test:

- build/run and login with an account that has one or more teams, and possibly some lists
- Go to the Reader
- Verify that the first four items in the tab bar are the default items, in this order
  - Following
  - Discover
  - Likes
  - Saved
- Verify that teams (and lists, if you have any) are displayed after the four default items and are listed in alphabetical order

#### Notes: 
1. in this implementation, the viewModel talks directly to CoreData and ReaderTopicService. I'm keen to abstract this part away, in a different object, but I also don't want to over-engineer the code. Reviewers please share your opinion.
2. At this stage, it is expected that some UI tests may fail. Feature flag will be disabled before merging into develop.

PR submission checklist:
